carservice/src/main/java/com/senlainc/warsaw/tyurin/entity/Order.java

```diff
@@ -16,17 +16,17 @@ public class Order {
     private long id;
     @Column(name = "price")
     private double price;
-    @Column(name = "submission_date")
+    @Column(name = "submissionDate")
```

Comment:
Why you decide to change naming in your DB? SQL had different naming convention then java. i thing previous was more suitable for DB

Reply:
It was bad idea I agree. Changed it back